webcomponents/tool/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
```

Comment:
tool/src/main/java/org/sakaiproject/webcomponents/permissions/PermissionsEntityProvider.java is the only java file in webcomponents? can it just be moved to entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/ ?

Reply:
Or better still, move it into the webapi

Reply:
More work though

Reply:
yup whichever easier ;)